optuna/storages/grpc/_auto_generated/api_pb2_grpc.py

```diff
@@ -0,0 +1,916 @@
+# mypy: ignore-errors
```

Comment:
I added this line to avoid the mypy errors.

Reply:
According to https://stackoverflow.com/questions/67130900/why-is-mypy-checking-files-that-ive-excluded, we can remove this line by adding some configuration in setup.cfg.